client/reader/stream/index.jsx

```diff
@@ -458,6 +549,16 @@ class ReaderStream extends Component {
 	}
 }
 
+/* eslint-disable */
+// wrapping with Main so that we can use withWidth helper to pass down whole width of Main
+const wrapWithMain = ( Component ) => ( props ) =>
+	(
+		<div>
```

Comment:
Did we mean to use `<Main />` here? I don't really understand the purpose of the component, could you elaborate? Can we move the wrapper just inside the rendered markup of the component?